docs/src/testing/gas-etimation.md

```diff
@@ -35,7 +38,30 @@ Multiplication of those values gives us a gas cost for each component:
 
 We should remember that only the most expensive factor will be taken into account, so our overall gas cost is `20.48`.
```

Comment:
`~21` maybe to be consistent with our way of displaying

Reply:
Thought about that, but in the note I've mentioned that it will be rounded up